### DISCUSSION TITLE
Section 4, Lesson 1 of Foundry Fundamentals - "make install" failing

### DISCUSSION THREAD

**Discussion by petkotodorov** [Cyfrin/foundry-full-course-cu#2550] [Q&A] [answered]
EDIT: Resolved! "make install" seems to be not needed, as the needed dependencies are committed in the repo already. "make build" worked fine and everything is in place.

Just started Section 4 of the Foundry Fundamentals. Made a clone of repo in a fresh directory and just typed "make install" as Patrick did, bit instead of a successful result, I receive the following error:

 
<img width="1376" alt="Screenshot 2024-09-25 at 16 30 08" src="https://github.com/user-attachments/assets/2e05ac3d-856a-4c94-868f-bc1faae68621">

No changes have beed done to the project after cloning..

**Comment by petkotodorov** [comment, marked as answer]
Resolved! "make install" seems to be not needed, as the needed dependencies are committed in the repo already. "make build" worked fine and everything is in place.

**Reply by EngrPips** [reply]
Interesting. I am glad you figured that out.
